### DISCUSSION TITLE
Converting HTTP targets to webhooks

### DISCUSSION THREAD

**Discussion by LivintLucian** [mozts2005/ZendeskApi_v2#541] [Q&A]
Hello guys,

I saw the announce for "The deprecation of HTTP targets and conversion to webhooks". I was wondering, from a developer perspective, if I'll should change anything in my code logic. I read in the article that webhooks are created and managed in Zendesk Admin Center. And wanna ask, ZendeskAPI will be changed as well? Or supposed to make changes only in Admin Center for this?

Sorry if the question is unclear, I'm newbie here. 

Thanks!

**Comment by mozts2005** [comment]
Can please share links to the pages you reference?

**Comment by LivintLucian** [comment]
Hi @mozts2005, sure, this is the link: https://support.zendesk.com/hc/en-us/articles/1260803041889
And the question which is unclear for my is this one:
`
If I create targets using the API, do I need to change my process?
Yes. You should start using the API to create equivalent webhooks rather than targets as soon as possible.
`
